src/app/pages/confirm-test-details/components/confirm-submit-modal/__mocks__/nav-params.mock.ts

```diff
@@ -0,0 +1,6 @@
+export class NavParamsMock {
```

Comment:
Can we move this to the ```mock/ionic-mocks``` folder which is at the same level as the src file, this mock might be useful elsewhere